myapp/app/helpers/tasks_helper.rb

```diff
@@ -3,6 +3,10 @@ def options_for_select_of_statuses
     Task.statuses.keys.map { |status| [I18n.t(status, scope: [:activerecord, :enums, :task, :status]), status] }
   end
 
+  def options_for_select_of_statuses_with_blank_option
+    options_for_select_of_statuses.unshift([I18n.t('form.unselected'), ''])
+  end
```

Comment:
You could show the unselected option with `include_blank: true`!

https://railsdoc.com/page/select_tag